hack/after-build/update-generated-docs.sh

```diff
@@ -39,8 +39,10 @@ shopt -u dotglob
 
 kube::util::gen-analytics "${KUBE_ROOT}"
 
+KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
```

Comment:
Is there a better place under hack/lib to set this once rather than in both *-generated-docs.sh scripts?